go/gas-oracle/gasprices/l2_gas_pricer_test.go

```diff
@@ -0,0 +1,171 @@
+package gasprices
+
+import (
+	"math"
+	"testing"
+)
+
+type CalcGasPriceTestCase struct {
+	name                     string
+	avgGasPerSecondLastEpoch float64
+	expectedNextGasPrice     float64
+}
+
+func returnConstFn(retVal float64) func() float64 {
+	return func() float64 { return retVal }
+}
+
+func runCalcGasPriceTests(gp GasPricer, tcs []CalcGasPriceTestCase, t *testing.T) {
+	for _, tc := range tcs {
+		nextEpochGasPrice, err := gp.CalcNextEpochGasPrice(tc.avgGasPerSecondLastEpoch)
+		if tc.expectedNextGasPrice != nextEpochGasPrice || err != nil {
+			t.Fatalf("failed on test: %s", tc.name)
+		}
+	}
+}
+
+func TestCalcGasPriceFarFromFloor(t *testing.T) {
+	gp := GasPricer{
+		curPrice:              100,
+		floorPrice:            1,
+		getTargetGasPerSecond: returnConstFn(10),
+		maxChangePerEpoch:     0.5,
```

Comment:
Change to either store internally as `%` -- or preferably remove the `%` entirely

Reply:
Change to `multiplier`

Reply:
I'm not 100% sure of what the intentions were here when this comment was made